packages/common/src/core/schemas/shared/HubItemEntitySchema.ts

```diff
@@ -0,0 +1,42 @@
+import { IConfigurationSchema } from "../types";
+import {
+  ENTITY_ACCESS_SCHEMA,
+  ENTITY_CATEGORIES_SCHEMA,
+  ENTITY_FEATURED_CONTENT_SCHEMA,
+  ENTITY_IMAGE_SCHEMA,
+  ENTITY_IS_DISCUSSABLE_SCHEMA,
+  ENTITY_LOCATION_SCHEMA,
+  ENTITY_NAME_SCHEMA,
+  ENTITY_SUMMARY_SCHEMA,
+  ENTITY_TAGS_SCHEMA,
+  ENTITY_TIMELINE_SCHEMA,
+} from "./subschemas";
+
+/**
+ * Defines a shared schema for an IHubItemEntity's editiable
+ * fields. All item entity schemas should "extend" this schema
```

Comment:
Maybe suggest looking at the Initiative or Project schema as an example... b/c you can't actually "extend" a const :)